src/test/groovy/com/synopsys/integration/blackduck/api/project/ProjectServiceTestIT.java

```diff
@@ -220,8 +220,8 @@ public void testCloning() throws Exception {
         assertNotNull(ProjectServiceTestIT.project);
         assertNotNull(projectVersionView);
 
-        String projectUrl = ProjectServiceTestIT.project.getHref().get();
-        String projectVersionUrl = projectVersionView.getHref().get();
+        HttpUrl projectUrl = ProjectServiceTestIT.project.getHref().get();
+        HttpUrl projectVersionUrl = projectVersionView.getHref().get();
```

Comment:
Another two Optionals without explicit acknowledgement of potential non-existence.